src/sc/src/services/public_api/create.rs

```diff
@@ -8,6 +10,7 @@ use fluvio_auth::AuthContext;
 use crate::services::auth::AuthServiceContext;
 
 /// Handler for create topic request
+#[instrument(skip(request, auth_context))]
```

Comment:
trace level

Reply:
This one is actually at the root of many requests, I think it would be valuable to keep